acceptance/pre-suite/01_install_rototiller.rb

```diff
@@ -8,8 +8,9 @@
   end
   scp_to(sut, gem_name, gem_name)
 
+  rake_version = `rake --version`.split[2]
   if ENV['RAKE_VER']
```

Comment:
it should probably check `if $(which rake)` or similar, but i dunno if i care enough to fix that for now.